CMakeLists.txt

```diff
@@ -431,6 +431,7 @@ SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 ##
 #############################################################
 if ("${CMAKE_GENERATOR}" STREQUAL "Xcode")
+	message(WARNING "(Using the XCODE Generator has previously yielded problems. See https://github.com/moves-rwth/storm/issues/65)")
```

Comment:
Yes, I thought it would be good to provide an alternative to building with XCode, for example using the commandline. We could alternative also refer to Clion.